cfy_manager/components/validations.py

```diff
@@ -680,6 +680,18 @@ def _validate_ldap_certificate_setting():
         )
 
 
+def _validate_crontab_accessible():
+    """Confirms that `cfyuser` can run crontab."""
+    proc = sudo(['-u', 'cfyuser', 'crontab', '-l'], ignore_failures=True)
```

Comment:
Probably best to try/except and then just output "Crontab is not usable by cfyuser: {err}", because we'll have problems if it fails for any other reason too.